daemon/execdriver/windows/exec.go

```diff
@@ -8,22 +8,16 @@ import (
 
 	"github.com/Sirupsen/logrus"
 	"github.com/docker/docker/daemon/execdriver"
-	"github.com/docker/docker/pkg/stringid"
```

Comment:
The changes in this file are part of the named pipe updates by @jstarks.  These are not required for the graphdriver refactor, just for testing on actual Windows.